cmd/dist/common.go

```diff
@@ -12,10 +17,31 @@ import (
 	"github.com/containerd/containerd/remotes"
 	"github.com/containerd/containerd/remotes/docker"
 	imagesservice "github.com/containerd/containerd/services/images"
+	"github.com/crosbymichael/console"
+	"github.com/pkg/errors"
 	"github.com/urfave/cli"
 	"google.golang.org/grpc"
 )
 
+var registryFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "insecure,k",
+		Usage: "Skip SSL certificate validation",
+	},
+	cli.BoolFlag{
+		Name:  "plain-http",
+		Usage: "Allow connections using plain HTTP",
+	},
+	cli.StringFlag{
+		Name:  "user",
```

Comment:
Since this can also include a password, shouldn't it be named `creds` or something similar?